### DISCUSSION TITLE
Getting "http2: Transport received Server's graceful shutdown GOAWAY" after switching to req

### DISCUSSION THREAD

**Discussion by bertold** [imroc/req#309] [General]
We recently switched from go-resty to req between microservices and we are seeing a HTTP/2 request failing intermittently (maybe 1 out of 1,000) with this error message. (We did not see this error ever with go-resty). Both microservices are built with Go. Has anybody seen this? Are there any settings, configuration options that would be worth trying? For the time being, retries have been added and we will see in a few hours if it resolves the issue. Thank you.

**Comment by 0xuLiang** [comment]
#491 #376
